docs/_releases/v0.5.15.md

```diff
@@ -21,12 +21,12 @@ tag: v0.5.15
 - [UI] Navigation Menu Icon Scale with zoom in/out  @Ayush-Khandelwal-007 (#2966)
 - Bump nokogiri from 1.11.1 to 1.11.5 in /docs @dependabot (#2987)
 
-## 🧰 Maintenance
+## Maintenance
 
 - [CI] Makefile: `make site` for conveniently building docs @leecalcote (#2993)
 - [CI] Helm: Test of helm chart releaser for remote repo @leecalcote (#2982)
 
-## 📖 Documentation
+## Documentation
```

Comment:
```suggestion
**Documentation**
```

Reply:
Yooo @alphaX86  👋🏻 
Can you please change these across all the files, All the other stuff LGTM 💯🎉